bandit.yaml

```diff
@@ -0,0 +1,3 @@
+[bandit]
```

Comment:
From a quick read, their isn't a difference between the .yaml and .yml file extension but for consistency sake maybe we should use `.yml`.